locust/render.py

```diff
@@ -311,7 +364,12 @@ def _enrich_changes(changes: List[Dict[str, Any]], root_url: str) -> None:
     "json": render_json,
     "yaml": render_yaml,
     "html": generate_render_html(html_file_section_handler_vanilla),
-    "html-github": generate_render_html(html_file_section_handler_github),
```

Comment:
I think it's okay to add more targets.

A few suggestions: let us add `github`, which provides the compressed GitHub rendering, `github-full`, and `html-full`.

`html` should produce the compressed HTML rendering.

We should keep `html-github` for backwards compatibility.